bravado/mapping/swagger20_validator.py

```diff
@@ -0,0 +1,39 @@
+from jsonschema import validators, _validators
+from jsonschema.exceptions import ValidationError
+from jsonschema.validators import Draft4Validator
+
+
```

Comment:
Ack. I will add a description. I would still want to separate it out from `validate.py` as `swagger20_validator` will grow much larger with many more validation customizations.